src/ui/RecipeTitleCard.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled';
+import Flex from './Flex';
+
+const StyledFooter = styled.footer`
+  display: flex;
+  gap: 8px;
+`;
+
+const BydelName = styled.footer`
```

Comment:
i don't think this should be a footer. there should only be one footer on a page. and i tend to think it should be the same on all pages - similar to a header.